OrbitCore/ElfFile.cpp

```diff
@@ -84,10 +90,7 @@ bool ElfFileImpl<ElfT>::GetFunctions(Pdb* pdb,
     }
 
     std::string name = symbol_ref.getName() ? symbol_ref.getName().get() : "";
-
-#if __linux__
-    name = LinuxUtils::Demangle(name.c_str());
-#endif
+    std::string pretty_name = llvm::demangle(name);
```

Comment:
This llvm::demangle is not part of llvm-8 (only llvm-9). vcpkg has only llvm-8. 
There is itaniumDemangle in llvm-8, which might be an option. Or with conan we can also have llvm-9

Reply:
This fails on my setup as well.

Reply:
I reverted this for now - seems like we need a wrapper functions for this - to detect LLVM version and act accordingly.